### PR TITLE
Update dependencies and bigint api

### PR DESCRIPTION
**Pull request by NikVolf**
related to #5667

### CODE REVIEW

**Comment by NikVolf** [comment]
waiting on https://github.com/debris/elastic-array/pull/4

**Comment by debris** [comment]
@NikVolf merged and released version `0.8`

**Comment by rphmeier** [comment]
The whole `bigint`/`ethcore-bigint` crate situation is confusing. We should break these into `parity-bigint` and `parity-fixedhash` and publish both on crates.io in separate repos along with RLP.

**Comment by NikVolf** [comment]
latest commit was reviewed in #5686

File: util/src/journaldb/earlymergedb.rs [hunk 1]
```diff
@@ -23,6 +23,7 @@ use memorydb::*;
 use super::{DB_PREFIX_LEN, LATEST_ERA_KEY};
 use super::traits::JournalDB;
 use kvdb::{KeyValueDB, DBTransaction};
+use heapsize::HeapSizeOf;
```
Comment (tomusdrw) [inline util/src/journaldb/earlymergedb.rs]: Shouldn't it be re-exported from `bigint`?

**Pull request merged**